spec/services/appointment_notification_service_spec.rb

```diff
@@ -76,5 +76,21 @@
       eligible_appointments = overdue_appointments.select { |a| a.communications.present? }
       expect(eligible_appointments.count).to eq(1)
     end
+
+    it 'should skip sending reminders to patients with just landline or invalid numbers' do
+        landline_number = create(:patient_phone_number, :landline)
```

Comment:
[STYLE] The assignments can become `let` block, will make the it block easier to read.